### PR TITLE
I Run Code

### PR DESCRIPTION
**Pull request by Defelo**
**Description**
Added I Run Code cog (#130)

### CODE REVIEW

File: morpheushelper/cogs/run_code.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from colours import Colours
+from emkc_api import Emkc, EmkcAPIException
+
+N_CHARS = 1000
+
+LANGUAGES = (
+    "awk,bash,brainfuck,c,cpp,crystal,csharp,d,dash,deno,elixir,emacs,go,haskell,java,jelly,julia,"
+    "kotlin,lisp,lua,nasm,nasm64,nim,node,osabie,paradoc,perl,php,python2,python3,ruby,rust,swift,"
+    "typescript,zig"
+).split(",")
```
Comment (Paul-Brauckmann) [inline morpheushelper/cogs/run_code.py:19]: Pls make a list out of this

Comment (use-to) [reply morpheushelper/cogs/run_code.py:19]: pls do it fast

Comment (Defelo) [reply morpheushelper/cogs/run_code.py:19]: > pls do it fast

thank you so much for this constructive comment

**Review by Paul-Brauckmann — DISMISSED** [review]

File: morpheushelper/cogs/run_code.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        self.bot = bot
+
+    @commands.command(usage=translations.run_usage)
+    @supported_languages_docs
+    async def run(self, ctx, *, args: str):
+        """
+        run some code
+        """
+
+        if not (match := re.fullmatch(r"```([a-zA-Z\d]+)\n(.+?)```", args, re.DOTALL)):
```
Comment (TheCataliasTNT2k) [inline morpheushelper/cogs/run_code.py:41]: Ignore needed ` so code can be run without it

**Review by TheCataliasTNT2k — CHANGES_REQUESTED** [review]

**Review by TheCataliasTNT2k — APPROVED** [review]

**Pull request merged**